docs/content/preview/explore/fault-tolerance/_index.md

```diff
@@ -17,7 +17,22 @@ menu:
 type: indexpage
 ---
 
-YugabyteDB can continuously serve requests in the event of planned or unplanned outages, such as system upgrades and outages related to a node, availability zone, or region. YugabyteDB's [High availability](../../architecture/core-functions/high-availability/) is achieved through a combination of distributed architecture, data replication, consensus algorithms, automatic rebalancing, and failure detection mechanisms, ensuring that the database remains available, consistent, and resilient to failures of fault domains.
+Resilience, in the context of cloud databases, refers to the ability to withstand and recover from various types of failures, ranging from hardware malfunctions and software bugs to network outages and natural disasters. A resilient database system is designed to maintain data integrity, accessibility, and continuity of operations, even in the face of adverse events. Achieving resilience in cloud databases requires a multi-faceted approach, involving robust architectural design, effective data replication and backup strategies, load balancing, failover mechanisms, and comprehensive monitoring and incident response procedures.
+
+YugabyteDB has been designed ground up to be resilient. YugabyteDB can continuously serve requests in the event of planned or unplanned outages, such as system upgrades and outages related to a node, availability zone, or region. YugabyteDB's [High availability](../../architecture/core-functions/high-availability/) is achieved through a combination of distributed architecture, data replication, consensus algorithms, automatic rebalancing, and failure detection mechanisms, ensuring that the database remains available, consistent, and resilient to failures of fault domains.
+
+In the following sections, explore the key strategies, technologies, and best practices for enhancing the resilience of cloud databases, enabling organizations to leverage the benefits of the cloud, while maintaining the highest levels of data protection and service availability.
+
+## Need for resilience
+
+The need for resilience in cloud databases arises from several factors:
+
+- **Distributed nature**: Cloud databases are often distributed across multiple physical locations, introducing complexities in data synchronization and potential points of failure.
+- **Ephemeral instances**: Virtual machines are spawned on commodity hardware which can disappear at any time. They are ephemeral.
+- **Unpredictable workloads**: Cloud databases must handle fluctuating and unpredictable workloads, which can strain resources and potentially lead to performance degradation or outages. High loads can cause power surges. Backup generators have failed too.
+- **Failure of fault domains**: Although public clouds have come a long way since the inception of AWS in 2006, region and zone outages are still fairly common, happening once or twice a year. Disk and machines commonly fail due to multiple reasons.
```

Comment:
```suggestion
- **Failure of fault domains**: Although public clouds have come a long way since the inception of AWS in 2006, region and zone outages are still fairly common, happening once or twice a year. Disks and machines commonly fail due to a variety of reasons.
```